doc/server_API.md

```diff
@@ -138,15 +125,14 @@ On success:
 
 ## get_module_all_whitelists
```

Comment:
What does this call mean? All whitelists that are bound to module? There's only one whitelist that is bound to each module.

Either delete this route  or make this into `GET /get_module_whitelist?unique_id=<...>` whichi will return whitelist name for that module or info if there's no whitelist bound to module.

Example result:
```json
{
    "whitelist_exists": true,
    "whitelist_name": "family"
}
```
```json
{
    "whitelist_exists": false
}
```

because module without bound whitelist is still a valid state in our database, I think json based response is better suited for error reporting than eror codes, but it is my opinion.